cli/wallet/validator.go

```diff
@@ -36,6 +39,10 @@ func newValidatorCommands() []*cli.Command {
 					Required: true,
 					Usage:    "Address to register",
 				},
+				&cli.BoolFlag{
```

Comment:
The idea of preserving backwards compatibility is good, but we need to make transfer-based registration a default behaviour. So let’s rename the flag and revert its meaning, something like useRegisterCall will suit.